user_docs/en/changes.md

```diff
@@ -17,6 +17,11 @@ The available options are:
 * The timeout to perform a multiple keypress is now configurable; this may be especially useful for people with dexterity impairment. (#11929, @CyrilleB79)
 * When performing a braille cursor routing action, NVDA can now automatically speak the character at the cursor. (#8072, @LeonarddeR)
   * This option is disabled by default. You can enable "Speak character when routing cursor in text" in NVDA's braille settings.
+* Improvements in Microsoft PowerPoint text objects:
```

Comment:
I think "objects" is slightly too technical a word here. Maybe "text boxes" instead?

Reply:
```suggestion
* Improvements when editing Microsoft PowerPoint text boxes:
```